packages/frontend/src/components/DataViz/config/DataVizAggregationConfig.tsx

```diff
@@ -72,40 +80,46 @@ export const DataVizAggregationConfig: FC<Props> = ({
     onChangeAggregation,
     aggregation,
 }) => {
+    const { colors } = useMantineTheme();
+    const { classes } = usePillSelectStyles({
+        backgroundColor: colors.indigo[0],
+        textColor: colors.indigo[4],
+    });
     const aggregationOptionsWithNone = options ?? [];
 
+    const selectOptions = aggregationOptionsWithNone.map((option) => ({
+        value: option,
+        label:
+            option === VizAggregationOptions.ANY
+                ? 'Any value'
+                : capitalize(option),
+    }));
+
     return (
-        <Select
-            radius="md"
-            data={aggregationOptionsWithNone.map((option) => ({
-                value: option,
-                label: capitalize(option),
-            }))}
-            itemComponent={AggregationItem}
-            icon={aggregation && <AggregationIcon aggregation={aggregation} />}
-            value={aggregation ?? aggregationOptionsWithNone?.[0]}
-            onChange={(value: VizAggregationOptions | null) =>
-                value && onChangeAggregation(value)
-            }
-            styles={(theme) => ({
-                input: {
-                    width: '110px',
-                    fontWeight: 500,
-                },
-                item: {
-                    '&[data-selected="true"]': {
-                        color: theme.colors.gray[7],
-                        fontWeight: 500,
-                        backgroundColor: theme.colors.gray[2],
-                    },
-                    '&[data-selected="true"]:hover': {
-                        backgroundColor: theme.colors.gray[3],
-                    },
-                    '&:hover': {
-                        backgroundColor: theme.colors.gray[1],
+        <Tooltip label="Aggregation type" variant="xs" withinPortal>
+            <Select
+                withinPortal
+                data={selectOptions}
+                itemComponent={AggregationItem}
+                value={aggregation ?? aggregationOptionsWithNone?.[0]}
+                onChange={(value: VizAggregationOptions | null) =>
+                    value && onChangeAggregation(value)
+                }
+                classNames={{
+                    item: classes.item,
+                    dropdown: classes.dropdown,
+                    input: classes.input,
+                    rightSection: classes.rightSection,
+                }}
+                styles={{
+                    input: {
```

Comment:
I can't remember: is the classNames styling no good when a parameter is required (aggregation here)? It's a little odd that we are stying input with styles and classnames. But maybe that's life in modern react 😂 . I'm sure this isn't the only place we are doing things like that.